centreon-certified/canopsis/canopsis2x-events-apiv2.lua

```diff
@@ -0,0 +1,890 @@
+#!/usr/bin/lua
+--------------------------------------------------------------------------------
+-- Centreon Broker Canopsis Connector Events
+--------------------------------------------------------------------------------
+
+
+-- Libraries
+local curl = require "cURL"
+local string = require "string"
+local sc_common = require("centreon-stream-connectors-lib.sc_common")
+local sc_logger = require("centreon-stream-connectors-lib.sc_logger")
+local sc_broker = require("centreon-stream-connectors-lib.sc_broker")
+local sc_event = require("centreon-stream-connectors-lib.sc_event")
+local sc_params = require("centreon-stream-connectors-lib.sc_params")
+local sc_macros = require("centreon-stream-connectors-lib.sc_macros")
+local sc_flush = require("centreon-stream-connectors-lib.sc_flush")
+
+--------------------------------------------------------------------------------
+-- Misc function
+--------------------------------------------------------------------------------
+
+function table.removekey(table, key)
```

Comment:
This is a function had been found using internet search to solve the problem to remove a key from my table in aim to send this key in another route (pbehavior comment route =/= pbehavior route). So in fact I really need to catch the removing key, not only to put it out of the table.
I'll put it locally and renaming it as you advised because it's true I wasn't thinking about the issue that you pointed out. But this may be a very specific problem and I'm not sure we'll need again to do this kind of action with table keys. So I'm not sure we really need to add this function (and think again its behavior with the removing key) in sc_common. (If you think it could happen again, feel free to tell me and we'll create a new enhancement ticket around it).